### DISCUSSION TITLE
Songfacts Plugin?

### DISCUSSION THREAD

**Discussion by Cameron-IPFSPodcasting** [strawberrymusicplayer/strawberry#1043] [Ideas]
When I use Strawberry, I always wish there were more song info about the currently playing track. I believe Clementine did a wiki search that *mostly* found info about the current band/song playing.

Is it possible to add song info to the Context tab/panel?

Songfacts claims to have an API, but I haven't looked any further than this... https://www.songfacts.com/blog/pages/songfacts-api

On a related note, is there documentation how to build a plugin that would do this? I could try to build something and submit a pull request, but honestly think the learning curve would be huge (for me), thus would take a very long time if I were to attempt such a thing.

Thanks,

Cameron

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
